sdk/turing/ensembler.py

```diff
@@ -395,7 +395,7 @@ def match_dependency(spec, name):
         # conda dependencies whose spec differs slightly from Python's setuptools.
         # We could install the complete conda library but this is too bulky if the goal is
         # to just carry out this matching.
-        return spec == name or re.match(f'{name}[><=\s]+', spec) is not None
+        return spec == name or re.match(name + r"[><=\s]+", spec) is not None
```

Comment:
Edited this to fix the deprecation warning about the use of `\s` as an escape sequence in f strings: https://github.com/gojek/turing/runs/7608305528?check_suite_focus=true#step:6:63